3.10/release-notes-upgrading-changes310.md

```diff
@@ -34,6 +34,29 @@ The fulltext index type is now deprecated in favor of [ArangoSearch](arangosearc
 Fulltext indexes are still usable in this version of ArangoDB, although their usage is
 now discouraged.
 
+### Geo indexes
+
+After an upgrade from a version below 3.10 to a version of 3.10 and
+higher, it is recommended to drop and recreate geo indexes. The reason is
+that GeoJSON polygons are interpreted slightly different (and more
+correctly) in the newer versions.
+
+Legacy geo indexes will continue to work and continue to produce the
+same results as in earlier versions, since they will have the flag
+`legacyPolygons` implicitly set to `true`.
+
+Newly created indexes will have `legacyPolygons` set to `false` by default
+and thus enable the new polygon parsing.
+
+Note that linear rings that are boundaries of polygons
```

Comment:
I'd completely break from explaining what we were doing before.
"Note that, from ArangoDB 3.10, **as per the standard**, linear rings [...] are not normalized automatically. The 'interior' of a polygon strictly conforms to the GeoJSON standard: ..."
"Note that this differs from the < 3.10 interpretation of GeoJSON polygons (and link to the old documentation)"

Reply:
Done.